Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -536,16 +536,14 @@ def customLogLevel(custom: acts.logging.Level = acts.logging.INFO):
                 )
             )
 
-    return s
```

Comment:
I think it might be better to return the value, even if we no longer recommend the caller *using* the value. Then people wouldn't have to update all their scripts immediately.

Better would be to add a comment in the docstring to say that the `Sequence` value is returned for compatibility reasons.

If this is adopted, then I don't know what is best for the function signature, which is mostly just for documentation purposes. (Do we ever check it?)

Reply:
I was also thinking about this but I fear if we do not make a clean cut no one will adopt. cc @paulgessinger

Reply:
I think people will follow your documentation, the examples, and especially our workshop, all of which show the new syntax. Returning an unused value is a common idiom, so I don't think it's a problem even if this never changes - at least until we need something else, which would anyway change the signature.

Reply:
agreed :+1: will adopt this tomorrow